requirements/requirements_ansible.txt

```diff
@@ -4,8 +4,8 @@
 #
 #    pip-compile --output-file requirements/requirements_ansible.txt requirements/requirements_ansible.in
 #
-adal==0.5.0               # via msrestazure
-appdirs==1.4.3            # via openstacksdk, os-client-config
+adal==1.2.1               # via msrestazure
+appdirs==1.4.3            # via openstacksdk
```

Comment:
out of curiosity, how did you manage to generate this file? In the AWX devel branch, the pip-compile tool isn't functional for Ansible requirements, as I found out trying to do https://github.com/ansible/awx/pull/3682

Reply:
PYCURL_SSL_BACKEND=openssl pip-compile ...